site2/docs/client-libraries-java.md

```diff
@@ -402,6 +403,25 @@ consumer.acknowledge(messages)
 >     .build();
 > ```
 
+### NegativeAckRedeliveryBackoff
+
+The `NegativeAckRedeliveryBackoff` is to introduce a redelivery backoff mechanism which we can achieve redelivery with different delays according to the redeliveryCount the message is retried. 
+
+```java
+Consumer consumer =  client.newConsumer()
+        .topic("my-topic")
+        .subscriptionName("my-subscription")
+        .negativeAckRedeliveryBackoff(NegativeAckRedeliveryExponentialBackoff.builder()
+                .minNackTimeMs(1000)
+                .maxNackTimeMs(60 * 1000)
+                .build())
+        .subscribe();
+```
+
+> Notice: 
+>   1. The negativeAckRedeliveryBackoff will not work with `consumer.negativeAcknowledge(MessageId messageId)` because we are not able to get the redelivery count from the message ID.
```

Comment:
```suggestion
>   - The `negativeAckRedeliveryBackoff` does not work with `consumer.negativeAcknowledge(MessageId messageId)` because you are not able to get the redelivery count from the message ID.
```